modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
Misunderstanding. I think that require 'vizkit/vizkittypelib' is the right require, so I was suggesting to change the CMake code to always (non-optionally) install into LIBDIR/<project_name>/